assets/js/voronoi.js

```diff
@@ -71,7 +82,7 @@ function handleFiles(file) {
 
 function voronoi() {
   const { width, height } = mainImageData;
-  const points = Array((mainImageData.data.length / (4 * 500)) | 0)
+  const points = Array((mainImageData.data.length / pointsToUse) | 0)
```

Comment:
Instead of doing it this way. I would just paste the number of points here without dividing or anything.
Just restrain the point to be between 1 and number of pixels divided by some number.